bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/handler/Atd100Handler.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.satel.internal.handler;
+
+import static org.openhab.binding.satel.internal.SatelBindingConstants.*;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.satel.internal.command.ReadZoneTemperature;
+import org.openhab.binding.satel.internal.command.SatelCommand;
+import org.openhab.binding.satel.internal.config.Atd100Config;
+import org.openhab.binding.satel.internal.event.SatelEvent;
+import org.openhab.binding.satel.internal.event.ZoneTemperatureEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link Atd100Handler} is responsible for handling commands, which are
+ * sent to one of the channels of a ATD-100 device.
+ *
+ * @author Krzysztof Goworek - Initial contribution
+ */
+public class Atd100Handler extends WirelessChannelsHandler {
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_ATD100);
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private ScheduledFuture<?> pollingJob;
+
+    public Atd100Handler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        if (pollingJob == null || pollingJob.isCancelled()) {
```

Comment:
FYI. Initialize should always update the pollingJob, also when `pollingJob` would be set. The `initialize()` method is called when the user has updated the refresh. So a new refresh should be set. However, in practise I don't think it will be a problem, because `dispose()` has always been called before `initialize()` is called in case of an configuration update.

Reply:
Thanks for the info.